core/config/docs/core.toml

```diff
@@ -669,3 +669,22 @@ TransmitQueueMaxSize = 10_000 # Default
 # when sending a message to the mercury server, before aborting and considering
 # the transmission to be failed.
 TransmitTimeout = "5s" # Default
+
+[Telemetry]
+# Enabled turns telemetry collection on or off.
+Enabled = false # Default
+# Endpoint of the OTEL Collector.
+Endpoint = 'example.com/collector' # Example
+# CACertFile is the file path of the TLS certificate used for secure communication with the OTEL Collector.
+# Required unless InescureConnection is true.
+CACertFile = 'cert-file' # Example
+# InsecureConnection bypasses the TLS CACertFile requirement and uses an insecure connection instead.
+# Only available in dev mode.
+InsecureConnection = false # Default
+# TraceSampleRatio is the rate at which to sample traces. Must be between 0 and 1.
+TraceSampleRatio = 0.01 # Default
```

Comment:
@pkcll does this make sense as a default? or should we choose a different value? (I don't recall where it comes from)

Reply:
1% seems reasonable default